data/transactions/logic/assembler.go

```diff
@@ -1996,7 +2002,7 @@ func (ops *OpStream) assemble(text string) error {
 	if ops.Errors != nil {
 		l := len(ops.Errors)
 		if l == 1 {
-			return errors.New("1 error")
+			return fmt.Errorf("1 error: %w", ops.Errors[0])
```

Comment:
A bunch of the tests look for error messages like `arg 0 wanted uint64 got []byte...`  but now with more specific types I've seen things like `arg 0 wanted uint64 got uint64` because the String method on the new StackType just returns the name which defaults to AVMType. 

I think it'll be a confusing error message unless I also include the bounds but it seems like it'd require a bunch of changes to the tests to allow the more descriptive type. 

wdyt?